src/main/java/Storage.java

```diff
@@ -0,0 +1,114 @@
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * @author bakwxh
+ * @version 0.1
+ */
+public class Storage {
+	/**
+	 * Path to save file.
+	 */
+	private String filePath;
+
+	/**
+	 * Constructor.
+	 * @param filePath Path to save file.
+	 */
+	public Storage(final String filePath) {
```

Comment:
Instead of using the final keyword in the parameter of the Storage constructor, would it be better to use the final keyword in the filePath instance field?